js/README.md

```diff
@@ -74,3 +74,19 @@ Build and test:
 yarn run build
 yarn run test
 ```
+
+## Release Notes
+
+## Version 1.5.0
```

Comment:
why moving from 1.3.0 to straight 1.5.0? what is the reasoning?

Reply:
the current version is 1.4.0 but it seems like we missed to update dev branch during last deployment.
The missing change is just to update the package version and the samples using that version.